index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Simple survey form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
```

Comment:
Please remove commented line in your codebase to make your code clean

Reply:
[OPTIONAL]: To make your PRs descriptions and review comments more human-friendly I would recommend using emojis 100 +1 technologist heavy_check_mark

> Please remove commented line in your codebase to make your code clean